CommonTools/ParticleFlow/plugins/PFnoPileUp.cc

```diff
@@ -0,0 +1,45 @@
+#include "CommonTools/ParticleFlow/plugins/PFnoPileUp.h"
```

Comment:
This is a new plugin: please merge the .h and .cc file
(Even if not newly introduced in this PR, you could also profit and do the same for PFPileUp)